reledmac.dtx

```diff
@@ -9680,6 +9799,12 @@
 %    \end{macrocode}
 % \end{macro}
 % \end{macro}
+% \begin{macro}{\iflinenumannotationotherside}
+% The \cs{iflinenumannotationotherside} boolean allows to print the annotation in the other side regarding the line number.
```

Comment:
"in the other side regarding the line number" → "on the side opposite the line number"